man/systemd.network.xml

```diff
@@ -244,6 +244,38 @@
           if <literal>RequiredForOnline=no</literal>.</para>
         </listitem>
       </varlistentry>
+      <varlistentry>
+        <term><varname>AdministrativeState=</varname></term>
+        <listitem>
+          <para>Takes a string indicating how networkd should manage the link administrative state.
+          Specifically, this controls how networkd controls the network device's <literal>IFF_UP</literal>
+          flag, which is sometimes controlled by system administrators by running e.g.,
+          <command>ip set dev eth0 up</command> or <command>ip set dev eth0 down</command>.</para>
+
+          <para>Takes one of <literal>start-on</literal>, <literal>always-on</literal>,
+          <literal>manual</literal>, <literal>always-off</literal>, or <literal>start-off</literal>.
+          When <literal>manual</literal>, networkd will not change the administrative state at all;
+          the system administrator must bring the interface administratively up or down, as desired.
+          When <literal>start-on</literal> or <literal>start-off</literal>, networkd will set the
+          administrative state on or off, respectively, when networkd starts, but it will not change
```

Comment:
ditto (and everywhere else too)

(Yeah, I know we are not very good at this right now, but let's try to get this right at least for new documentation)